README.md

```diff
@@ -78,7 +78,7 @@ If you use the Habitat platform in your research, please cite the [Habitat 1.0](
       ```bash
       git clone --branch stable https://github.com/facebookresearch/habitat-lab.git
       cd habitat-lab
-      pip install -e habitat-lab  # install habitat_lab
+      pip install -e .  # install habitat_lab
```

Comment:
@dhruvbatra I think this is actually correct, `cd habitat-lab` refers to the root repository that was just cloned. `pip install -e habitat-lab` refers to the sub directory `habitat-lab/habitat-lab` that contains the `setup.py` file.
This is due to the separation of habitat-lab and habitat-baselines into separate packages

Reply:
Sorry for the delay. I thought I was trying to fix this error reported by a user: https://github.com/facebookresearch/habitat-lab/issues/993 

But I suspect they were using an older branch, so yes, we can close this now.